doc/services/pm/device.rst

```diff
@@ -158,8 +157,64 @@ a particular device. It is important to emphasize that, although the
 state is tracked by the subsystem, it is the responsibility of each device driver
 to handle device actions(:c:enum:`pm_device_action`) which change device state.
 
-Each :c:enum:`pm_device_action` have a direct an unambiguous relationship with
-a :c:enum:`pm_device_state`.
+Each :c:enum:`pm_device_state` state is defined as follows:
+
+- ``OFF``: Device hardware is not powered. This is the initial state from which
+  a device driver is initialized.
+
+  When a device driver is initialized, we do not know the state of the device.
+  As a result, the :c:enum:`pm_device_action` ``TURN_ON`` action should be able
+  to transition the device from any internal state into ``SUSPENDED``, since no
+  guarantees can be made across resets. This is typically achieved through
+  toggling a reset pin or triggering a software reset through a register write
+  before performing any additional configuration needed to meet the requirements
+  of ``SUSPENDED``. For devices where this is not possible, the device driver
+  must presume the device is in either the ``OFF`` or ``SUSPENDED`` state at
+  time of initialization, as these are the states within which device drivers
+  may be deinitialized.
+
+  If a device has been specified as this device's power domain, and said device
+  becomes needed by the system, the power domain device will be transitioned
```

Comment:
If the device becomes needed by the system, then shouldn't the device get transitioned to `ACTIVE` state as well?

Reply:
_the device_ here being the the device **on** the power domain right? The power domain device can become needed because some other device sharing the same power domain device became needed, _the device_ would only be moved all the way to ACTIVE if it became needed :)